gke-scripts/TrustConfig.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: security.cloud.google.com/v1alpha1
+kind: TrustConfig
+metadata:
+  name: default
+spec:
+  # You must include a trustStores entry for the trust domain that
+  # your cluster is enrolled in.
+  trustStores:
+  - trustDomain: ${PROJECT_ID}.svc.id.goog
+    # Trust identities in this trustDomain if they appear in a certificate
+    # that chains up to this root CA.
+    trustAnchors:
+    - certificateAuthorityServiceURI: ${ROOT_CA_URI}
```

Comment:
It's definition in `00-common-env.sh` is the old beta API of CAS. Can we update that to reflect the new GA API?

Reply:
What is the new value?

Reply:
`//privateca.googleapis.com/projects/PROJECT_ID/locations/ROOT_CA_POOL_LOCATION/caPools/ROOT_CA_POOL_NAME`

Reply:
Done.